dapp/js/containers/Pages/CampaignDeployer/Component.jsx

```diff
@@ -28,6 +28,16 @@ export default class Deployer extends Component {
 
   componentDidMount() {
     this.props.setAccount(web3.eth.accounts[0]);
+    //set default account to user current account if they are not set
+    let campaignValues = Object.assign({}, this.props.campaignValues);
+    let accountTypes = ['escapeCaller', 'securityGuard', 'donor', 'recipient']
+    for(let i=0; i<accountTypes.length; i++) {
+        if(!campaignValues[accountTypes[i]]){
+            campaignValues[accountTypes[i]] = web3.eth.accounts[0];
+        }
+    }
+    this.props.updateCampaignValues(campaignValues);
```

Comment:
I see... Thank you. I will just accept it for now. However, it would be much more correct to either set the accounts before by listening to WEB3_NEWSTATE action in the reducer.

I think the whole campaign deployer should actually store everything in the state rather than calling action each time there is new user input. But that is something I am planning to do...